bebraven/platform
# Authenticate AWS S3 bucket requests for Rise360 content

### Description
Task: https://app.asana.com/0/1174274412967132/1198943778694978

We already have AWS access keys configured that are used when publishing Rise360 content using the SDK gem. This just piggybacks off the SDK to make the proxied requests for showing the Rise360 content use authenticated presigned URLs. After deploying this, turn the bucket permissions for both dev and prod to only allow the IAM accounts we want to be able to access the buckets.

### Motivation and Context
Prevents our content from being available to the world.

### Test Plan
- Make sure you can hit a URL like: https://platformweb/rise360_module_versions/9?state=some_state before this change and it works
- Merge this change and make sure it still works
- Make sure a URL like https://our-s3-bucket.s3.amazonaws.com/lessons/adsjlhbvi3njl1mau14uea2bdhre/index.html works with the bucket still not locked down
- Turn on permissions for the bucket and make sure the above URL doesn't work (no access error)
- Finally make sure the URL in step 1 still works after locking the bucket down

### Types of changes




### Checklist:
